OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -1281,8 +1291,14 @@ function Install-OctopusDeploy {
         Write-Log "Configuring Octopus Deploy instance to use free license ..."
         $args += @('--free')
     } else {
-        Write-Log "Configuring Octopus Deploy instance to use supplied license ..."
-        $args += @('--licenseBase64', $licenseKey, '--skipLicenseCheck')
+        if (Test-OctopusVersionSupportSkipLicenseCheck) {
```

Comment:
Instead of the `if/else`, you could just have a subsequent if that checks if its supported, and only append it then.